doc/man3/SHA256_Init.pod

```diff
@@ -95,11 +95,12 @@ ANSI X9.30
 
 =head1 SEE ALSO
 
+L<EVP_Q_digest(3)>,
 L<EVP_DigestInit(3)>
 
 =head1 HISTORY
 
-All of these functions were deprecated in OpenSSL 3.0.
+All of these functions except SHA*() were deprecated in OpenSSL 3.0.
```

Comment:
It should mention which ones were changed to macros here.

Reply:
Done